### PR TITLE
Allow TexturePacker's JSON-Hash

### PR DESCRIPTION
**Pull request by MSGhero**
Export options in TP for json include JSON-Array and JSON-Hash, with hash being more useful imo.

No breaking changes.

### CODE REVIEW

File: flixel/graphics/frames/FlxAtlasFrames.hx [hunk 1]
```diff
@@ -53,39 +53,56 @@ class FlxAtlasFrames extends FlxFramesCollection
...
-				angle = FlxFrameAngle.ANGLE_NEG_90;
+				texturePackerHelper(frame.filename, frame, frames);
 			}
-			else
+		}
+		
+		// JSON-Hash
+		else
+		{
+			for (frameName in Reflect.fields(data.frames))
```
Comment (Gama11) [inline flixel/graphics/frames/FlxAtlasFrames.hx]: Is Reflection necessary here? I.e., couldn't we parse this with a typedef?

**Comment by Gama11** [comment]
Would be nice to have a small unit test for this, espeically since you probably have some test for this already anyway.

File: flixel/graphics/frames/FlxAtlasFrames.hx (hunk 1 above)
Comment (Gama11) [inline flixel/graphics/frames/FlxAtlasFrames.hx]: On second thought, I guess we're already working with Dynamic anyway... (which isn't ideal to begin with).

**Comment by MSGhero** [comment]
Idk what the deal is with the test package, but it's not recognizing the files I put into the assets folder even if I add an assets path in project.xml. I got around the json files by pasting their data directly, and I just set the bitmapdata to a non-null one so it doesn't error from being null.

**Comment by Gama11** [comment]
Yeah, there are several issues with using assets in the unit tests... I usually used workarounds similar to yours.

Thanks for the test!

**Pull request merged**